### PR TITLE
rint: further doc tweaks

### PR DESCRIPTION
**Pull request by RalfJung**
r? @nagisa

### CODE REVIEW

File: library/core/src/intrinsics.rs [hunk 1]
```diff
@@ -1787,8 +1787,9 @@ extern "rust-intrinsic" {
     /// so this rounds half-way cases to the number with an even least significant digit.
     ///
     /// May raise an inexact floating-point exception if the argument is not an integer.
-    /// However, Rust assumes floating-point exceptions cannot be observed, so this is not something that
-    /// can actually be used from Rust code.
+    /// However, Rust assumes floating-point exceptions cannot be observed, so these exceptions
+    /// cannot actually be utilized from Rust code.
+    /// In other words, this itnrinsic is equivalent in behavior to `nearbyintf32` and `roundevenf32`.
```
Comment (CryZe) [inline library/core/src/intrinsics.rs:1792]:
```suggestion
    /// In other words, this intrinsic is equivalent in behavior to `nearbyintf32` and `roundevenf32`.
```

File: library/core/src/intrinsics.rs [hunk 2]
```diff
@@ -1798,8 +1799,9 @@ extern "rust-intrinsic" {
     /// so this rounds half-way cases to the number with an even least significant digit.
     ///
     /// May raise an inexact floating-point exception if the argument is not an integer.
-    /// However, Rust assumes floating-point exceptions cannot be observed, so this is not something that
-    /// can actually be used from Rust code.
+    /// However, Rust assumes floating-point exceptions cannot be observed, so these exceptions
+    /// cannot actually be utilized from Rust code.
+    /// In other words, this itnrinsic is equivalent in behavior to `nearbyintf64` and `roundevenf64`.
```
Comment (CryZe) [inline library/core/src/intrinsics.rs:1804]:
```suggestion
    /// In other words, this intrinsic is equivalent in behavior to `nearbyintf64` and `roundevenf64`.
```

File: library/core/src/intrinsics.rs (hunk 1 above)
Comment (RalfJung) [reply library/core/src/intrinsics.rs]: *oops* good catch, thanks

**Comment by nagisa** [comment]
This is great, thank you for taking time to react to the comment!

@bors r+ rollup

**Pull request merged**